internal/middleware/user.go

```diff
@@ -8,18 +8,27 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/hashicorp/golang-lru/v2/expirable"
 	"github.com/rs/zerolog/log"
+	"github.com/ucl-arc-tre/portal/internal/config"
 	"github.com/ucl-arc-tre/portal/internal/graceful"
 	"github.com/ucl-arc-tre/portal/internal/rbac"
 	"github.com/ucl-arc-tre/portal/internal/types"
 	"gorm.io/gorm"
 )
 
 const (
-	userCacheTTL      = 1 * time.Hour
 	userContextKey    = "user"
 	usernameHeaderKey = "X-Forwarded-Preferred-Username" // See: https://oauth2-proxy.github.io/oauth2-proxy/configuration/overview
 )
 
+func getUserCacheTTL() time.Duration {
```

Comment:
I’m not a fan of this - dev should, in my opinion, be as close to prod as possible. (sorry not looked at the rest of the PR) why by adding this is the user no-longer cacheable?

Reply:
It's back to how it was now.